src/utils/globals.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+// eslint-disable-next-line import/prefer-default-export
+export function setElementsDebugInfo() {
```

Comment:
You could consider exporting this as part of an object to avoid the eslint rule above.